lib/components/chat_panel.dart

```diff
@@ -38,6 +38,31 @@ class _ChatPanelWidgetState extends State<ChatPanelWidget> {
     });
   }
 
+  Widget cntTextStyle(cnt) {
```

Comment:
Can you move the changes in this file out of this PR since this is the other one?